app/src/main/java/org/mozilla/focus/utils/UrlUtils.java

```diff
@@ -82,38 +84,32 @@ public static String getSearchTermsOrUrl(Context context, String url) {
     }
 
     /**
-     * @return If a url matches the form of a first search URL for the current default search engine.
+     * @return The detected search engine or null if one was not matched with the URL
      */
-    private static boolean isDefaultSearchUrl(Context context, String urlSearch) {
+    private static SearchEngine isDefaultSearchUrl(String urlSearch) {
```

Comment:
nit: I expect methods that start with "is" to always return a boolean. Maybe we should rename this method now.